src/vm/arm/armsinglestepper.cpp

```diff
@@ -1051,6 +1051,18 @@ bool ArmSingleStepper::TryEmulate(T_CONTEXT *pCtx, WORD opcode1, WORD opcode2, b
 
             fEmulated = true;
         }
+        else if ((opcode1 & 0xff00) == 0x4400)
+        {
+            // A8.8.6 ADD (register, Thumb)
+            DWORD Rm = BitExtract(opcode1, 6, 3);
+            fEmulated = (Rm == 15);
```

Comment:
Nothing else in this file sets `fEmulated` like this. Would it be better to follow the established pattern?

Reply:
fEmulated seems to be set to true in under all other cases, here you only set it to emulated only if the source register is PC, which matches the comment on the function. Nonetheless, you seem to always modify Rd. I think it'd be safe to follow convention here.